modules/local/ancestry/extract_database.nf

```diff
@@ -17,18 +17,27 @@ process EXTRACT_DATABASE {
 
     output:
     tuple val(meta38), path("GRCh38_*_ALL.pgen"), path("GRCh38_*_ALL.psam"), path("GRCh38_*_ALL.pvar.zst"), emit: grch38, optional: true
-    tuple val(meta38), path("deg2_hg38.king.cutoff.out.id"), emit: grch38_king, optional: true
+    tuple val(meta38), path("GRCh38_*.king.cutoff.out.id"), emit: grch38_king, optional: true
     tuple val(meta37), path("GRCh37_*_ALL.pgen"), path("GRCh37_*_ALL.psam"), path("GRCh37_*_ALL.pvar.zst"), emit: grch37, optional: true
-    tuple val(meta37), path("deg2_phase3.king.cutoff.out.id"), emit: grch37_king, optional: true
+    tuple val(meta37), path("GRCh37_*.king.cutoff.out.id"), emit: grch37_king, optional: true
     path "versions.yml", emit: versions
 
     script:
     meta38 = ['build': 'GRCh38']
     meta37 = ['build': 'GRCh37']
-    def king = params.target_build == "GRCh37" ? "deg2_phase3.king.cutoff.out.id" : "deg2_hg38.king.cutoff.out.id"
 
     """
-    tar -xvf $reference --wildcards "${params.target_build}*" $king
+    tar -xf $reference --wildcards "${params.target_build}*" meta.txt 2> /dev/null
+
+    DB_VERSION=\$(cat meta.txt)
+
+    if [ "\$DB_VERSION" != "2.0.0-alpha.3" ]; then
```

Comment:
@nebfield don't you think this is too strict? It will involve us updating the reference panels on the FTP every software release. Perhaps we can version reference panels on their own, and only bump it when we change some of the code that would change them>